### DISCUSSION TITLE
Enable comments for NextJs app

### DISCUSSION THREAD

**Discussion by Patriksafar** [markdoc/markdoc#409] [Q&A] [answered]
Hey 👋 ,

I am using Markdoc within Nextjs application. In my `next.config.js` I have `withMarkdoc` but I am not able to pass `allowComments: true` as a option. 

Is there a way how to do enable it for NextJs setup? 

I have checked docs but I haven't found a way how to do so. 

I would expect something like this which is not possible 🤔 

```js
const withMarkdoc = require("@markdoc/next.js");

module.exports = withMarkdoc({ mode: "static" })({
  pageExtensions: ["js", "jsx", "ts", "tsx", "md", "mdoc"],
  allowComments: true,
});
```

**Comment by mfix-stripe** [comment, marked as answer]
Hey @Patriksafar, you can pass the `allowComments` option like this:

```js
withMarkdoc({ mode: "static", tokenizerOptions: {allowComments: true} })
```

**Reply by Patriksafar** [reply]
Works correctly! Thank you @mfix-stripe! 

Suggestion: Would be possible add it to docs for Markdoc?

**Reply by mfix-stripe** [reply]
Adding here: https://github.com/markdoc/docs/pull/109

Thanks @Patriksafar!
